### ISSUE TITLE
source code

### ISSUE THREAD

**Issue by lyllover** [ChunyuLiu188/Image-semantic-block-transmission-with-REINFORCE#1]
May I ask if you have the source code for the article Task-Oriented Image Transmission for Scene Classification in Unmanned Aerial Systems?

**Comment by ChunyuLiu188** [comment]
I'm sorry, but I don't have the source code for the article "Task-Oriented Image Transmission for Scene Classification in Unmanned Aerial Systems".

**Comment by lyllover** [comment]
Thank you for the reply! I would like to ask if your code is referencing this article?

**Comment by ChunyuLiu188** [comment]
Yes. However, the denoising network is simpler than the article. Additionally, the variance of the Gaussian noise in the channel is not modified with input and remains set at 0.01. Furthermore, the 8-bit quantization considered in the original article is not included. These aspects can be improved upon.

**Comment by lyllover** [comment]
Thanks so much for the inspire！
